applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -583,6 +588,70 @@ class EigensolverStrategy
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief Recover the solution related to the slave dofs in case of master-slave constraints.
+     * @details This function is an adaptation of the implementation in ConstraintUtilities,
+     *  since there the variables are assumed to be stored in SolutionStepValue.
+     *  Beware that this implementation is only valid for Block B&S, since the master-slave constraints
+     *  don't work with Elimination B&S yet.
+     */
+    void ReconstructSolution(
```

Comment:
```suggestion
    void ReconstructSlaveSolution(
```

I think this is a more applicable name